js/bybit.js

```diff
@@ -2933,19 +2954,37 @@ module.exports = class bybit extends Exchange {
         if (price !== undefined) {
             request['price'] = parseFloat (this.priceToPrecision (symbol, price));
         }
-        const stopPx = this.safeValue2 (params, 'stop_px', 'stopPrice');
+        const triggerPrice = this.safeValue2 (params, 'stop_px', 'stopPrice');
```

Comment:
Shouldn't we accept 'triggerPrice' here as well?

Reply:
I'm not sure, but we unified stopPrice and sometimes the exchange specific param for stopPrice is triggerPrice, in this case its stop_px.

If I were to add 'triggerPrice' here, what would be the best approach? can I have three params in safeValue2? should it be safeValue with the argument being a list of all three potential params, or is there something else I can use in this situation?